__init__.py

```diff
@@ -129,6 +129,29 @@ def currency_filter(num, grouping=True):
         return locale.currency(num, grouping=grouping)
 
 
+def _unique(values):
+    ret = []
+    for value in values:
+        if value not in ret:
+            ret.append(value)
+    return ret
```

Comment:
No--since python sets are unordered, casting to `set` (and then back to `list`) discards order information, which we want to keep here.